godtools/ViewControllers/Tract/TractViewController.swift

```diff
@@ -0,0 +1,68 @@
+//
```

Comment:
This looks great, I was going to tell you to use the parent method ` displayScreenTitle()` but this is a special case, maybe this viewcontroller will requiere changes during the development